README.md

```diff
@@ -23,6 +23,7 @@ Example:
 
 ```yaml
 title: "protolambda's Greeter fork"  # Define the HTML page title
+logo: "./logo.png"
```

Comment:
Would omit the dot here, and make it absolute: it's the img link as it appears in the html, not a linux filepath

Reply:
Fixed :)